hydra-tui/test/Hydra/TUI/OptionsSpec.hs

```diff
@@ -1,17 +1,17 @@
 module Hydra.TUI.OptionsSpec where
 
-import Hydra.Prelude
-import Test.Hydra.Prelude
+import           Hydra.Prelude
+import           Test.Hydra.Prelude
 
-import Hydra.Ledger.Cardano (NetworkId (Mainnet, Testnet), NetworkMagic (NetworkMagic))
-import Hydra.Network (Host (Host))
-import Hydra.TUI.Options (
-  parseCardanoNetworkId,
-  parseCardanoNodeSocket,
-  parseCardanoVerificationKey,
-  parseNodeHost,
- )
-import Options.Applicative (Parser, ParserResult (Success), defaultPrefs, execParserPure, info)
+import           Hydra.Ledger.Cardano (NetworkId (Mainnet, Testnet),
+                                       NetworkMagic (NetworkMagic))
+import           Hydra.Network        (Host (Host))
+import           Hydra.TUI.Options    (parseCardanoNetworkId,
+                                       parseCardanoNodeSocket,
+                                       parseCardanoVerificationKey,
+                                       parseNodeHost)
+import           Options.Applicative  (Parser, ParserResult (Success),
+                                       defaultPrefs, execParserPure, info)
```

Comment:
@KtorZ this doesn't look like our fourmolu config?

Reply:
That's quite strange indeed 🤔